### DISCUSSION TITLE
The 3d gun printing community need .zip file upload

### DISCUSSION THREAD

**Discussion by thanksmia** [mastodon/mastodon#16181] [General]
Is it possible to enable .zip file upload ? for  printing files  as a whole?

**Comment by trwnh** [comment]
You may as well enable arbitrary file uploads at that point, no? At which point, you may as well just use a file hosting service...

**Reply by thanksmia** [reply]
where the f is the decentralized file upload on activitypub or mastodon? something like bittorent  but via buttholes over  decentralized toilets .
